packages/admin-panel/src/api/TupaiaApi.js

```diff
@@ -120,6 +120,9 @@ export class TupaiaApi {
     }
 
     const responseBlob = await response.blob();
+    if (returnBlob) {
```

Comment:
This is so that the blob can be loaded into the table, instead of saved to the user's device